apis/user_search.yaml

```
@@ -0,0 +1,852 @@
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError'
+  /catalog/v1/users?domain={domain}:
+    get:
+      summary: "Find users by domain"
+      description: >
+        To search for all users on your authenticating domain, know that users are only
+        linked to a domain if they have been given a role on the domain. Thus, this search will find
+        all users who have a role on the domain.
+
+
+        Please read the "Domain specific params" on the [overview page](#?route=overview) to avoid common errors.
+
+        #### Examples
+          - `?domain=data.ny.gov`
+
+      parameters:
+        - $ref: '#/components/parameters/domain'
+      responses:
+        "200":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SearchResponse'
+        "401":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError'
+  /catalog/v1/users?roles={role}&domain={domain}:
+    get:
+      summary: "Find users by role"
+      description: >
+        Users may be assigned [roles](https://support.socrata.com/hc/en-us/articles/202950278-Understanding-user-roles)
+        within each domain. The role names can vary by domain, but the standard set shown in the linked
+        article is supported. To find users with a given role on a given domain, use the  `roles` and `domain` params.
+
+
+        Please read the "Domain specific params" on the [overview page](#?route=overview) to avoid common errors.
+
+        #### Examples
+          - `?roles=publisher&domain=data.ny.gov`
+          - `?roles=publisher&roles=editor&domain=data.ny.gov`
+          - `?roles=publisher,editor&domain=data.ny.gov`
+      parameters:
+        - $ref: '#/components/parameters/roles'
+        - $ref: '#/components/parameters/domain'
+      responses:
+        "200":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SearchResponse'
+        "400":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequestError'
+        "401":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError'
+  /catalog/v1/users?team_ids={id}&domain={domain}:
+    get:
+      summary: "Find users by team_id"
+      description: >
+        Users may belong to one or more
+        [teams](https://support.socrata.com/hc/en-us/articles/360012053653-Create-and-Manage-Teams#h_01H475EQ1ASTT15A8YE1QYXFXG)
+        on a domain. Teams are uniquely identified by a four-by-four like that used with the
+        [`ids` param](#?route=get-/catalog/v1/users-ids--4x4-).
```

```suggestion
        [`ids` param](#?route=cmp--parameters-ids).
```
does it make more sense to link this to the new params page for this?

---

no. what I was trying to do was redefine what a 4x4 is - in case this is the first page a user sees that term.   I removed the link and just copied the same text out of the ids route.